python/lsst/dax/obscore_generator/gen_img_obscore.py

```diff
@@ -20,20 +20,19 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import json
+import math
 import os
-import uuid
 import urllib.parse as UrlParser
-import math
-import pandas as pd
-import json
+import uuid
 
-# import cProfile
 import click
-
+import lsst.geom as geom
+import pandas as pd
 from lsst.daf.base.dateTime.dateTime import DateTime
-from lsst.obs.base import Instrument
 from lsst.daf.butler import Butler
-import lsst.geom as geom
+from lsst.obs.base import Instrument
```

Comment:
Now `lsst.pipe.base.Instrument` -- we don't want an obs_base dependency we don't need.